src/main/java/hudson/plugins/git/extensions/impl/PruneStaleTag.java

```diff
@@ -64,17 +67,45 @@ public void decorateFetchCommand(GitSCM scm,
                                      GitClient git,
                                      TaskListener listener,
                                      FetchCommand cmd) throws IOException, InterruptedException, GitException {
-        listener.getLogger().println("Pruning obsolete local tags");
+        listener.getLogger().println("Pruning obsolete local tags that does not exists on remotes");
```

Comment:
Minor grammar change
```suggestion
        listener.getLogger().println("Pruning obsolete local tags that do not exist on remotes");
```